scripts/common/info-view.ts

```diff
@@ -0,0 +1,105 @@
+import {displayWarning} from './uiElements'
+
+// /**
+//  * Displays information about the Venmo transaction and generates the Venmo deep link.
+//  * @param {int} price - The price of the ticket to display.
+//  * @param {string} venmoRecipient - The reciepient of the venmo transaction to display.
+//  */
+// const generateVenmoInfo = function generateVenmoInfoForView(price:number, venmoRecipient:string) {
+//     const venmoLinkButton = document.querySelector('.open-venmo') as HTMLButtonElement;
+//     const venmoLinkPath = `venmo://paycharge?txn=pay&recipients=${venmoRecipient}&note=Eweek Banquet`;
+  
+//     venmoLinkButton.onclick = function openVenmoOnClick() {
+//       if (/Android|webOS|iPhone|iPad|iPod|Opera Mini/i.test(navigator.userAgent)) {
+//         window.open(venmoLinkPath, '_blank');
+//       } else {
+//         window.open('https://venmo.com', '_blank');
+//       }
+//     };
+  
+//     const DOMTicketPrice = document.querySelector('.ticket-price');
+//     const DOMVenmoRecipient = document.querySelector('.venmo-recipient');
+  
+//     DOMTicketPrice.textContent = price;
+//     DOMVenmoRecipient.textContent = venmoRecipient;
+//   };
+  
+ /**
+   * Displays various information about the event details in the form.
+   * @param {string} description - The description of the event to display.
+   * @param {int} price - The price of the ticket to display.
+   * @param {string} venmoRecipient - The reciepient of the venmo transaction to display.
+   */
+  const displayInfo = function displayInfoForView(description:string, venmoRecipient:string) {
```

Comment:
Add space after the type colon for each param.

Example: `description: string`